src/commands/scanner/run.ts

```diff
@@ -60,8 +60,7 @@ export default class Run extends ScannerCommand {
 			char: 't',
 			description: messages.getMessage('flags.targetDescription'),
 			longDescription: messages.getMessage('flags.targetDescriptionLong'),
-			// If you're specifying local files, it doesn't make much sense to let you specify anything else.
```

Comment:
Not directly tied to this change, but I was in this file for the first version that used a separate flag. Removed the `org` and `file` flags. Made --target required. This allows us to fall back to the default toolbelt validations instead of a custom message.